pytest_splunk_addon/standard_lib/data_models/Alerts.json

```diff
@@ -1,75 +1,128 @@
-    "objects":[
-    {
-        "name": "Alerts",
-        "tags": [
-            ["alert"]
-        ],
-        "fields_cluster": [],
-        "search_constraints": "tag=alert",
-        "fields": [
-            {
-                "name": "app",
-                "type": "required",
-                "comment":"The application involved in the event, such as win:app:trendmicro, vmware, nagios."
-            },
-            {
-                "name": "body",
-                "type": "required",
-                "comment":"The body of a message." 
-            },
-            {
-                "name": "dest",
-                "type": "required",
-                "comment":"The destination of the alert message, such as an email address or SNMP trap. You can alias this from more specific fields, such as dest_host, dest_ip, or dest_name."                
-            },
-            {
-                "name": "id",
-                "type": "required",
-                "comment":"The unique identifier of a message." 
-            },
-            {
-                "name": "severity",
-                "type": "required",
-                "expected_values": [
-                    "critical",
-                    "high",
-                    "medium",
-                    "low",
-                    "informational"
-                  ],
-                "comment":"The severity of a message." 
-            },
-            {
-                "name": "severity_id",
-                "type": "optional",
-                "comment":"A numeric severity indicator for a message." 
-            },
-            {
-                "name": "src",
-                "type": "optional",
-                "comment":"The source of the message. You can alias this from more specific fields, such as src_host, src_ip, or src_name." 
-            },
-            {
-                "name": "subject",
-                "type": "optional",
-                "comment":"The message subject." 
-            },
-            {
-                "name": "type",
-                "type": "required",
-                "expected_values":[
-                    "alarm",
-                    "alert",
-                    "event",
-                    "task",
-                    "warning"
-                ],
-                "comment":"The message type." 
-            }
-        ],
-        "child_dataset": []
-    }]
-}
\ No newline at end of file
+    "objects": [
+        {
+            "name": "Alerts",
+            "tags": [
+                [
+                    "alert"
+                ]
+            ],
+            "fields_cluster": [],
+            "search_constraints": "tag=alert",
+            "fields": [
+                {
+                    "name": "app",
+                    "type": "required",
+                    "comment": "The application involved in the event, such as win:app:trendmicro, vmware, nagios."
+                },
+                {
+                    "name": "body",
```

Comment:
This is deprecated in CIM documentation, how can it be required for PSA?

Reply:
body field marked as required in cim doc.

Reply:
Just a side note. We should consult the CIM *.jsons, not the CIM Docs. The CIM Docs are not the source of truth.

Reply:
@alexeisuv I referred both the things docs and CIM jsons, but for required fields if we check the jsons only 2-3 fields are required across all models in CIM *.json, so then the meaning of running tests using PSA for CIM compatibility would diminish in my opinion, as PSA only runs tests for required or conditional fields.